Image Build error

When selecting `Build extra binary images` in `menuconfig`
the build fails when creating the minix 720k image:

```
install /Users/helge/src/mellvik/elks/elks/arch/i86/boot/Image /Users/helge/src/mellvik/elks/target/linux
rm -f /Users/helge/src/mellvik/elks/image/fd720-minix.bin
mfs  /Users/helge/src/mellvik/elks/image/fd720-minix.bin genfs -1 -n14 -i192 -s720 /Users/helge/src/mellvik/elks/target
No free slots in bitmap found
make[2]: *** [minixfs] Error 255
make[1]: *** [fd720-minix] Error 2
```

--- a/elkscmd/Applications
+++ b/elkscmd/Applications
@@ -45,7 +45,7 @@ sys_utils/getty			:boot	:sysutil
 sys_utils/login			:boot	:sysutil
 ash/ash		::bin/sh	:boot	:ash	# install as /bin/sh
 #sash/sash	::bin/sh	:boot	:~ash	# install as /bin/sh if no ASH
-sash/sash				:boot	:sash
+sash/sash				:1440k	:sash
 sys_utils/mount			:boot	:sysutil
 sys_utils/umount		:boot	:sysutil
 sys_utils/clock			:boot	:sysutil

